realm/test-gitConnector.json

```diff
@@ -0,0 +1,8 @@
+{

```

Comment:
do not put test json files in this repo
remove the file

Reply:
extra json file removed